packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -81,7 +81,10 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
             currency
         },
         throwOnBadResponse: true,
-        fetchOptions
+        fetchOptions: {
+            ...fetchOptions,
+            credentials: 'omit'
```

Comment:
Hello @alexvuong , you shouldn't have to overwrite "credentials: omit", it is that way by default in commerce-sdk-isomorphic v.1.10.1 (https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/27609f35def0494db7ec45fb948a18c0eac2d25e/src/static/clientConfig.ts#L71)